File: kotlin-ide/j2k/new/testData/newJ2k/issues/kt-19348.java
public class TestMutltipleCtorsWithJavadoc {
    private String x;
    private String y;

    // ---
    // Constructors
    //

    /**
     * Javadoc for 1st ctor
     * @param x
     */
    public TestMutltipleCtorsWithJavadoc(String x) {
        this.x = x;
    }

    /**
     * Javadoc for 2nd ctor
     * @param x
     * @param y
     */
    public TestMutltipleCtorsWithJavadoc(String x, String y) {
        this(x);
        this.y = y;
    }
}

File: nj2k/testData/newJ2k/issues/kt-19348.kt
class TestMutltipleCtorsWithJavadoc
/**
 * Javadoc for 1st ctor
 * @param x
 */(private val x: String) {
    private var y: String? = null

    // ---
    // Constructors
    //

    /**
     * Javadoc for 2nd ctor
     * @param x
     * @param y
     */
    constructor(x: String, y: String?) : this(x) {
        this.y = y
    }


}